test-support/src/main/java/org/springframework/ldap/test/unboundid/EmbeddedLdapServerFactoryBean.java

```diff
@@ -47,8 +47,14 @@ public void setPort(int port) {
 	}
 
 	@Override
-	protected EmbeddedLdapServer createInstance() throws Exception {
-		return EmbeddedLdapServer.newEmbeddedServer(this.partitionName, this.partitionSuffix, this.port);
+	protected EmbeddedLdapServer createInstance() {
```

Comment:
Because this method can be overridden by subclasses, we can't narrow the allowed exceptions thrown by the method.